Prace_domowe/Praca_domowa1/Grupa1/Dawid_Janus/PD1_Dawid_Janus.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>Pomysł na analizę był ciekawy jednak problemem jest wybór typu wykresu - jest dużo słupków przez co nakładają się i różnią się szerokością. Wynika to z tego, że temperatura miała wartości rzeczywiste a nie całkowitoliczbowe.</p><p>Lepsze rozwiązania:</p><ul><li>zastosowanie wykresu punktowego </li><li>pogrupowanie temperatur w pewne przedziały wartości i dla nich policzenie średniej, wówczas mógłby być wykres słupkowy</li><li>stosując podział temperatur z punktu 2 (o ile nie będzie dużo kategorii) można narysować boxploty</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>